src/app/(explore)/[...slug]/page.tsx

```diff
@@ -290,7 +290,7 @@ const page = ({ params }: { params: { slug: string[] } }) => {
           </div>
 
           {isRoot ? (
-            <div className='flex flex-col gap-6 h-full pb-8 overflow-scroll'>
+            <div className='flex flex-col gap-6 h-full pb-8 overflow-auto'>
```

Comment:
The scroll bars are still present in this page (I don't think I noticed this in my initial QA pass):

https://registry-git-fork-0tuedon-fix-satsie-qa-btc-knowledge-base.vercel.app/edgedevplusplus

![Screenshot from 2025-06-04 12-10-42](https://github.com/user-attachments/assets/7ecb8417-f349-4389-bf9d-08782d3dad51)